src/components/Dialog-Window/Tabs/DialogTabs.tsx

```diff
@@ -21,6 +21,58 @@ type Translation = {
   text: string;
 };
 
+const getItemInfo = (item: CommonItemType): {
+  hasText: boolean;
+  hasLinks: boolean;
+  hasVideo: boolean;
+  hasAudio: boolean;
+  showNote: boolean;
+  showTabs: boolean;
+} => {
+  const { description, topicImage, dialog } = item;
+
+  const smallScreen = useMedia('(max-width: 768px)');
+  const hasNote = (item.dialog?.hasNote && smallScreen) ?? false;
+  const showNote = hasNote && smallScreen;
+
+  const hasText = (dialog?.text || topicImage || description) ? true : false;
+  const hasLinks =
+    ((dialog?.links &&
+      dialog?.links?.filter((link) => Boolean(link.url)).length > 0) ||
+      dialog?.showAddLinks) ? true : false;
+  const hasVideo = (dialog?.video?.[0]?.path) ? true : false;
+  const hasAudio = (dialog?.audio?.audioFile?.[0]?.path) ? true : false;
+
+  var content = 0;
+  if (hasText) {
+    content++; 
+  }
+  if (hasLinks) {
+    content++; 
+  }
+  if (hasVideo) {
+    content++; 
+  }
+  if (hasAudio) {
+    content++; 
+  }
+  if (showNote) {
+    content++; 
+  }
```

Comment:
:)

```suggestion
  const numberOfContentItems = [hasText, hasLinks, hasVideo, hasAudio, showNote].filter(Boolean).length;
```

ooooor if we play JavaScript:

```suggestion
  const content = hasText + hasLinks + hasVideo + hasAudio + showNote;
```

Reply:
Nice!